test/features/steps/crud_database.py

```diff
@@ -85,6 +93,16 @@ def step_see_db_dropped(context):
     wrappers.expect_exact(context, 'Query OK, 0 rows affected', timeout=2)
 
 
+@then('we see database dropped and no default database')
+def step_see_db_dropped_no_default(context):
+    """Wait to see drop database output."""
+    user = context.conf['user']
+    host = context.conf['host']
+    context.currentdb = '(none)'
```

Comment:
Ah, my problem was I was relying on `after_scenario` to test the prompt (thus needing to set the current db to `(none)` since that's what the prompt displays.

Now that I noticed you can do `context.atprompt = True`, this should be more appropriate.